extensions-core/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureClientFactory.java

```diff
@@ -46,31 +48,20 @@ public AzureClientFactory(AzureAccountConfig config)
   }
 
   // It's okay to store clients in a map here because all the configs for specifying azure retries are static, and there are only 2 of them.
```

Comment:
Is it still true that there will only be at most 2 entries per storage account? Seems like with the addition of the retryCount in the key, this constraint has changed? Or what limits it to only 2 per account given this?

Reply:
there are only 2 places where retryCount is configured and they're both static configs so there's only ever 2 values for retryCount